### PR TITLE
Feat: chat header

### PR DESCRIPTION
**Pull request by alamorre**
Peep the custom styles example, looking minty

Should be an easy one to review!!

### CODE REVIEW

File: src/ChatFeed/ChatHeader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <Row
+      className="ce-chat-title"
+      style={{ ...styles.titleSection, ...style.titleSection }}
+    >
+      <Col
+        xs={2}
+        sm={0}
+        style={{
+          ...styles.mobileOptiom,
+          ...{ left: '6px' },
```
Comment (dhavelock) [inline src/ChatFeed/ChatHeader/index.tsx:26]: could we just put `...{ left: '6px' },` in the default styles?

File: src/ChatFeed/ChatHeader/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  subtitleText?: Properties;
+}
+
+export const styles = {
+  titleSection: {
+    width: '100%',
+    zIndex: 1,
+    backgroundColor: 'rgb(256, 256, 256, 0.92)',
+  } as Properties,
+  mobileOptiom: {
```
Comment (dhavelock) [inline src/ChatFeed/ChatHeader/styles.tsx:17]: typo- `mobileOption` I'm guessing?

**Review by dhavelock — APPROVED** [review]
lgtm just two comments

File: src/ChatFeed/ChatHeader/index.tsx (hunk 1 above)
Comment (alamorre) [reply src/ChatFeed/ChatHeader/index.tsx:26]: Yes! Adding

Comment (alamorre) [reply src/ChatFeed/ChatHeader/index.tsx:26]: Actually no. `mobileOptiom` is used on the left and right of the page and need this positioning different for each

- Default
- Usage Specific
- Prop Override

Its still in line with that

File: src/ChatFeed/ChatHeader/styles.tsx (hunk 2 above)
Comment (alamorre) [reply src/ChatFeed/ChatHeader/styles.tsx:17]: Yep! Changed

**Review by dhavelock — APPROVED** [review]

**Pull request merged**